frontend/js/src/components/Navbar.tsx

```diff
@@ -35,18 +35,15 @@ function Navbar() {
 
   return (
     <nav role="navigation">
-      <div className="navbar-header">
+      <div className="navbar-light navbar-header">
         <button
           type="button"
-          className="navbar-toggle collapsed"
+          className="navbar-toggler"
```

Comment:
The toggler styles need to be reviewed as they currently make it pretty small on mobile devices compared to existing website.

Also the toggler button seems to be in fixed position (scroll the page down, it stays on screen) which wasn't the case before

Reply:
Fixed